core/io/image.cpp

```diff
@@ -2600,12 +2601,12 @@ Ref<Image> Image::load_from_file(const String &p_path) {
 	return image;
 }
 
-Error Image::save_png(const String &p_path) const {
+Error Image::save_png(const String &p_path, bool speed_over_compression) const {
```

Comment:
We need to keep this one works the same as before, we can add a new function `save_png_fast` or `set_png_compression`.